### DISCUSSION TITLE
Scoreboard

### DISCUSSION THREAD

**Discussion by iCrafterMC** [Zrips/CMI#7730] [Feature Requests]
Add Scoreboard Feature with scoreboard.yml and /scoreboard edit (scoreboard id) for Chat editor

**Comment by ghost** [comment]
I hope too.

**Comment by CloudeLecaw** [comment]
i would like this too.
pls post suggestions next time here
https://github.com/Zrips/CMI/discussions

or add you suggestion here:
https://github.com/Zrips/CMI/discussions/2883
or here
https://github.com/Zrips/CMI/discussions/6623

you see the suggestion also allready exist.
mostly it worths more to add a reply to an allready existing discussion :-)

**Comment by Angelius782** [comment]
yes it would be wonderful ... since simplescore is over and not maintain....... :(

**Comment by CloudeLecaw** [comment]
pls close this or move it to discussion
